Artifacts/windows-powershellcore/Artifactfile.json

```diff
@@ -0,0 +1,31 @@
+{
+    "$schema": "https://raw.githubusercontent.com/Azure/azure-devtestlab/master/schemas/2016-11-28/dtlArtifacts.json",
+    "title": "PowerShell Core a.k.a PowerShell 6",
+    "description": "Installs PowerShell Core on Windows",
+    "publisher": "PowerShell Magazine",
+    "tags": [
+        "Windows",
+        "PowerShell",
+        "PowerShell Core",
+        "PowerShell 6"
+    ],
+    "iconUri": "https://raw.githubusercontent.com/rchaganti/azure-devtestlab/master/Artifacts/windows-powershellcore/Powershell_black_64.png",
+    "targetOsType": "Windows",
+    "parameters": {
+        "packageUrl": {
+            "type": "string",
+            "displayName": "Package URL",
+            "allowEmpty": false,
+            "description": "The PowerShell core package (MSI) URL from https://github.com/PowerShell/PowerShell/releases."
+        },
+        "installCRuntime": {
+            "type": "bool",
+            "displayName": "Install C runtime for Windows OS prior to Windows Server 2016?",
+            "allowEmpty": false,
+            "description": "Installing C runtime is a prerequisite for using PowerShell Remoting over WinRM in PowerShell core."
+        }      
+    },    
+    "runCommand": {
+        "commandToExecute": "[concat('powershell.exe -ExecutionPolicy bypass \"& ./installPowerShellCore.ps1',' -Verbose -packageUrl ', parameters('packageUrl'),' -installCRuntime $', parameters('installCRuntime'), '\"')]"
```

Comment:
You are not specifying `[CmdletBinding()]` in your PS1, so the parameter `-Verbose` will not have the desired effect.